src/include/sof/audio/multiband_drc/multiband_drc.h

```diff
@@ -0,0 +1,84 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2020 Google LLC. All rights reserved.
+ *
+ * Author: Pin-chih Lin <johnylin@google.com>
+ */
+#ifndef __SOF_AUDIO_MULTIBAND_DRC_MULTIBAND_DRC_H__
+#define __SOF_AUDIO_MULTIBAND_DRC_MULTIBAND_DRC_H__
+
+#include <stdint.h>
+#include <sof/audio/crossover/crossover.h>
+#include <sof/audio/drc/drc.h>
+#include <sof/platform.h>
+#include <sof/math/iir_df2t.h>
+#include <user/multiband_drc.h>
+
+struct comp_dev;
```

Comment:
i think we are missing a header if you are having to declare this.

Reply:
isn't this forward declaration? and we have included <sof/audio/component.h> in multiband_drc.c

Reply:
Ah I think we should just include the header here then. Unless we have some reason not to?

Reply:
Yeah I think you are right